docs/arlas-security.md

```diff
@@ -0,0 +1,107 @@
+# ARLAS security
+
+This page describes how to configure ARLAS in order to control access to resources and data.
+
+## Authentication
+Authentication is the process of identifying a user and is a prerequisite to [Authorisation](#authorisation).  
+Depending on the use case, some configuration must be set. Refer to [ARLAS server authentication configuration](arlas-server-configuration.md)
+The following use cases are illustrated with the ARLAS-Server service but are valid for all other ARLAS as ARLAS-persistence back-ends.
+
+* Use case 1: public access, no endpoint is protected (e.g. dev, test deployment)
+  - ARLAS server:  set `arlas_auth.enabled` to `false`
+  - ARLAS wui: set [authentication.use_authent](http://docs.arlas.io/arlas-tech/current/arlas-wui-configuration/) to `false` in `settings.yaml`. 
+
+* Use case 2: public access, some endpoints are protected (e.g. demo, freemium deployment)
+  - ARLAS server:  set `arlas_auth.enabled` to `true`
+  - ARLAS server:  set `arlas_auth.certificate_url` to the url of the jwks.json description of your identity provider  specified by [RFC7519](https://tools.ietf.org/html/rfc7519)
+  - ARLAS server:  set `arlas_auth.public_uris` to the needed value, 
+  e.g. `swagger.*:*,explore/.*:*` will only allow public access to URIs `/swagger.*` and 
+  `/explore/.*`
+  - ARLAS wui: set [authentication.use_authent](http://docs.arlas.io/arlas-tech/current/arlas-wui-configuration/) to `false` or `true` in `settings.yaml`, it depends on whether ARLAS wui must access protected end-points or not.
+
+* Use case 3: protected access (e.g. customer deployment)
+  - ARLAS server:  set `arlas_auth.enabled` to `true`
```

Comment:
Just to avoid repeating 'ARLAS-server' we can put

- ARLAS server:  
  * set `arlas_auth.enabled` to `true`
  * set `arlas_auth.certificate_url` to the url of the jwks.json description of your identity provider  specified by [RFC7519](https://tools.ietf.org/html/rfc7519)
  * set `arlas_auth.public_uris` to the needed value

Reply:
done